routes/chorals.js

```diff
@@ -38,18 +37,26 @@ router.get('/', function(req, res, next) {
   });
 });
 
-router.post('/', function(req, res, next) {
+function getChildrenFromRequest(req){
+  var reqChildren = req.body['children[]'];
+  // if the value is a string then convert it to an array
+  // for some reason when only one thing comes in it comes in as a
+  // string instead of an array
+  if (typeof reqChildren === 'string' || reqChildren instanceof String){
+      reqChildren = [ reqChildren ];
+  }
+  var children = [];
+  if(reqChildren && reqChildren.length){
+      children = reqChildren.map((val) => {return { _id: val } } );
+  }
+  return children;
+}
+
+router.post('/new', function(req, res, next) {
```

Comment:
To me it makes more sense to post to new. I don't really get why it was posting to nowhere in the first place